starter-code/pizza.js

```diff
@@ -1 +1,83 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function(){
```

Comment:
remember that you don't need all your functions inside of the $(document).ready(function()). You only need to put what you wan't loaded when the page load in that function.